docs/GuideTVOS.md

```diff
@@ -0,0 +1,89 @@
+---
+id: guide-tvos
+title: Apple TV changes
+layout: docs
+category: Guides (Apple TV)
```

Comment:
This should result in a new "Guides (Apple TV)" section being added to the sidebar. That's fine. Did you build and run the website and verified the sidebar is generated correctly? I don't foresee any issues here but just want to be sure.